packages/admin-rte/src/core/Rte.tsx

```diff
@@ -86,13 +86,21 @@ export interface CometAdminRteThemeProps {
         buttonBorderHover?: React.CSSProperties["color"];
         buttonBorderDisabled?: React.CSSProperties["color"];
     };
+    editor?: {
+        minHeight?: number;
+    };
 }
 
-export interface IProps extends CometAdminRteThemeProps {
+export interface IProps {
     value: EditorState;
     onChange: OnEditorStateChangeFn;
     options?: IOptions;
     disabled?: boolean;
+    theme?: CometAdminRteThemeProps;
```

Comment:
I put all theme-props  into a separate property, to improve the structure.

Reply:
How is this implemented for other components: do we have a way to customize theme (colors etc) using props?

If not, does it really make sense to have it for this component?

Reply:
It's probably really better to directly change the minHeight without involving the theme, for the sake of simplicity. 

> do we have a way to customize theme (colors etc) using props?

No, this does not exist.